src/d3a/models/strategy/__init__.py

```diff
@@ -185,6 +185,7 @@ def __init__(self):
         super(BaseStrategy, self).__init__()
         self.offers = Offers(self)
         self.enabled = True
+        self._allowed_disable_events = [AreaEvent.ACTIVATE, MarketEvent.TRADE]
```

Comment:
Very minor, but a name like "_allowed_events_when_disabled" would be more descriptive, IMO. What do you think?

Reply:
Will do, thanks!

Reply:
A bit out of of context but aren'T we sticking to small letters for [channel_names](https://github.com/gridsingularity/d3a/pull/675/files#diff-5c5fbb99e0155bbce18f8ca72347fd58R200)

Reply:
These are not channel names, but event enums.

Reply:
Sorry for not clear in first place, I meant about the link I used in my former comment